c7n/utils.py

```diff
@@ -42,6 +42,12 @@ def ignore_aliases(self, data):
 
 log = logging.getLogger('custodian.utils')
 
+try:
+    from yamlinclude import YamlIncludeConstructor
```

Comment:
its about 10 lines to do an include addition, picking up a third party dep has more cost than that. 

historically (yaml include prs from before), there's a separate question on doing this in yaml versus in app logic, via includes as a structure that allows for filtering and tracking location (relative files on values_from etc).

Reply:
Generally speaking, I'd like to keep my PRs as they are unless necessary, so that it won't create conflicts for the old team to merge because I've already moved on. Please feel free to close the PR if it is not desirable. Thanks.

Reply:
related PR https://github.com/cloud-custodian/cloud-custodian/pull/2184